src/components/DatePicker/datePickerChangelogData.ts

```diff
@@ -9,6 +9,15 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "2023-11-28",
```

Comment:
Please use `"Prerelease"` for the `date` value as well.  That final value will get set when we prep for the actual release.

Reply:
Done! Merging now.